### ISSUE TITLE
Getting npm output even with -q option

### ISSUE THREAD

**Issue by orsifacundo** [awslabs/automated-security-helper#32]
When running the scan an output similar to this is being shown:

```
added 1 package, and audited 3 packages in 761ms

found 0 vulnerabilities
npm notice 
npm notice New major version of npm available! 8.6.0 -> 10.2.5
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.2.5>
npm notice Run `npm install -g npm@10.2.5` to update!
npm notice 
```

Even when using the --quiet (-q) option

**Comment by orsifacundo** [comment]
adding the --quiet option to npm install seems to do the trick.. will further investigate.

**Comment by scrthq** [comment]
hey @orsifacundo - this may be fixed with my recent merged PR #34 ; would you mind testing to see if this is still ongoing when using the latest `main` branch content?

**Comment by orsifacundo** [comment]
Totally, it's fixed now. Thanks @scrthq.. closing this issue.
